apps/verification/src/verification.service.ts

```diff
@@ -182,44 +182,24 @@ export class VerificationService {
       const payload: IProofRequestPayload = {
         orgId: requestProof.orgId,
         url,
-        proofRequestPayload: {}
-      };
-
-      const proofRequestPayload = {
-        comment,
-        connectionId: requestProof.connectionId,
-        autoAcceptProof: requestProof.autoAcceptProof ? requestProof.autoAcceptProof : AutoAccept.Never,
-        goalCode: requestProof.goalCode || undefined,
-        parentThreadId: requestProof.parentThreadId || undefined,
-        willConfirm: requestProof.willConfirm || undefined
+        proofRequestPayload: {
+          comment,
+          connectionId: requestProof.connectionId,
+          autoAcceptProof: requestProof.autoAcceptProof ? requestProof.autoAcceptProof : AutoAccept.NEVER,
+          goalCode: requestProof.goalCode || undefined,
+          parentThreadId: requestProof.parentThreadId || undefined,
+          willConfirm: requestProof.willConfirm || undefined,
```

Comment:
What is the use of `willConfirm`?

Reply:
willConfirm states if the receiver of this connection will be sending an ack.